backend/models/listing.js

```diff
@@ -23,13 +23,14 @@ const ListingSchema = new mongoose.Schema(
     location: {
       address: {
         type: String,
-        required: true,
       },
-      latitude: {
-        type: Number,
-      },
-      longitude: {
-        type: Number,
+      latlng: {
```

Comment:
If we change the listing model here, we need to make sure that all the other places in the code that make a listing object are updated as well.